backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4333,3 +4333,45 @@ databaseChangeLog:
               - column:
                   name: test_performed_code
                   type: text
+  - changeSet:
+      id: create-test-performed-table
+      author: boban@skylight.digital
+      comment: creates a table to store the test performed code between device type and supported disease
+      changes:
+        - tagDatabase:
+            tag: create-test-performed-table
+        - createTable:
+            tableName: test_performed
```

Comment:
nit: I would name this something more descriptive like `test_performed_loinc_code`

Reply:
`device_test_performed_loinc_code`?